src/OWML.Utils/JsonHelper.cs

```diff
@@ -17,14 +17,14 @@ public static T LoadJsonObject<T>(string path)
 				.Replace("\\\\", "/")
 				.Replace("\\", "/");
 
-			try
-			{
-				return JsonConvert.DeserializeObject<T>(json);
-			}
-			catch (Exception)
-			{
-				return default;
-			}
+			//try
+			//{
+			return JsonConvert.DeserializeObject<T>(json);
+			//}
+			//catch (Exception)
+			//{
+			//	return default;
+			//}
```

Comment:
:/

Reply:
this isn't a release candidate :P

Reply:
there's a thousand comments we need to fix, wanna merge this before we get lots of conflicts when working in parallell ;)

Reply:
fixed this one ;)